packages/cli/commons/src/preconditions/isWebAuthenticated.ts

```diff
@@ -0,0 +1,17 @@
+import {Config} from '../config/config';
+import globalConfig from '../config/globalConfig';
+import {PreconditionError} from '../errors/preconditionError';
+
+const PRECONDITION_ERROR_CATEGORY = 'Wrong Authentication Type';
+
+export function IsWebAuthenticated() {
+  return async function () {
+    const config = new Config(globalConfig.get().configDir).get();
+    if (config.anonymous) {
+      throw new PreconditionError(
+        'This command does not support token authentication. Please authenticate again using coveo auth:login',
```

Comment:
`token authentication` is not specific enough.

```suggestion
        'This command does not support API key authentication. Please authenticate again using coveo auth:login',
```

Reply:
Done